spark/spark-spa/src/main/resources/velocity/layouts/component_layout.vm

```diff
@@ -50,7 +50,9 @@
       data-version-copyright-year="${currentVersion.copyrightYear()}"
       data-version-formatted="${currentVersion.formatted()}"
       data-version-full="${currentVersion.fullVersion()}"
-      data-is-server-in-drain-mode="${drainModeService.isDrainMode()}">
+      data-is-server-in-drain-mode="${drainModeService.isDrainMode()}"
+      data-meta="${meta}"
```

Comment:
this might need some escaping, esp if `meta` contains quotes, or other HTML chars that can possibly break.